tests/unit/FirebaseArray.spec.js

```diff
@@ -40,13 +40,20 @@ describe('$firebaseArray', function () {
       $utils = $firebaseUtils;
       $rootScope = _$rootScope_;
       $q = _$q_;
-      tick = function (cb) {
+
+      firebase.database.enableLogging(function () {tick()});
+      tick = function () {
         setTimeout(function() {
           $q.defer();
           $rootScope.$digest();
-          cb && cb();
-        }, 1000)
+          try {
```

Comment:
Hm. We have a flushAll() method that handles try/catch/$timeout.flush() already. Not a big deal if we like that or not, but consistency would be good here.